docs/guides/context.md

```diff
@@ -188,6 +188,21 @@ The `meta` object contains:
 - `state` - the current state in a normal transition (`undefined` for the initial state transition)
 - `action` - the assign action
 
+::: warning
```

Comment:
now we'll have two warnings stacked on top of each other, maybe worth combining them into one?

Reply:
Nah, they should be seen as separate warnings.